src/styles/_card.scss

```diff
@@ -0,0 +1,113 @@
+@import 'variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $width-card;
+  color: $color-background;
+  padding: $padding-card;
+  border: 1px solid $color-background;
```

Comment:
The `border` property uses `$color-background`, which might not be suitable if `$color-background` is meant for background colors. Consider using a more appropriate color variable for the border.